### ISSUE TITLE
Fex support

### ISSUE THREAD

**Issue by harelh** [CiscoDevNet/terraform-provider-aci#76]
Hi,
I didn't find any way to attach a FEX to ACI fabric with Terrform, It seems few component are missing (like FEX Profile..).
I made it meanwhile with the following steps & API calls :
Leaf 103 - Eth 1/1 - Fex 113
```

resource "aci_rest" "rest_fex_profile" {
  path       = "/api/node/mo/uni/infra/fexprof-FexP113.json"
  class_name = "infraFexP"
  content = {
    "name" = "FexP113"
  }
}

resource "aci_rest" "rest_infraFexBndlGrp" {
  path       = "/api/node/mo/uni/infra/fexprof-FexP113/fexbundle-FexP113.json"
  class_name = "infraFexBndlGrp"
  content = {
    "name" = "FexP113"
  }
}

resource "aci_access_port_selector" "fooaccess_port_selector" {
  leaf_interface_profile_dn = "uni/infra/accportprof-103"
  name                      = "Fex113"
  access_port_selector_type = "range"
  relation_infra_rs_acc_base_grp = "fexprof-FexP113"
}

resource "aci_access_port_block" "fooaccess_port_block" {
  access_port_selector_dn = aci_access_port_selector.fooaccess_port_selector.id
  name                    = "Fex113"
  from_card               = "1"
  from_port               = "1"
  to_card                 = "1"
  to_port                 = "1"

}

resource "aci_rest" "infraRsAccBaseGrp" {
  path       = "/api/node/mo/uni/infra/accportprof-103/hports-Fex113-typ-range.json"
  class_name = "infraRsAccBaseGrp"
  content = {
    "tDn": aci_rest.rest_infraFexBndlGrp.id
    "fexId": "113"
  }
}
```

Is there any option to add FEX support to the provide?

**Comment by dannielshalev** [comment]
Hi,
This is a very important component for us well 
We use Fex Extention a lot more common then we use leaves.

**Comment by nkatarmal-crest** [comment]
This issue has been fixed in the version 0.5.0 of the provider. Please try that and let me know if we can close this issue or not.
